.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug in syzkaller
+about: Create a report about a specific problem in syzkaller
+title: <EDIT ME> (try to follow "pkg/name: short description" format)
+labels: 'bug'
```

Comment:
We can use this to assign some labels like `needs-triage` or `new-issue`.

Reply:
A common practice is to have sheriff check:
https://github.com/dvyukov/syzkaller/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage
that's the list of issues that need to be acted on: ask for more information, close, confirm. Once it's confirmed by a sheriff, `needs-triage` is removed and some other labels may be added (`need-help`, `urgent`, etc).

Reply:
Aha! It is clear now. Agree, it is a good one to have.